### ISSUE TITLE
Panel title translations not loaded until the panel is loaded

### ISSUE THREAD

**Issue by roblourens** [microsoft/vscode#115112]
- Change the display language, restart
- See mixed english/non-english strings in pane titles;
- They change when I click on them
- But this only happens once, then they survive reloading the window

![Jan-26-2021 10-03-16](https://user-images.githubusercontent.com/323878/105885480-06f3d480-5fbe-11eb-9edc-9f41c6123891.gif)

**Comment by roblourens** [comment]
When I changed the language from chinese to spanish:

![image](https://user-images.githubusercontent.com/323878/105885710-54704180-5fbe-11eb-9753-b6505f672749.png)

**Comment by sbatten** [comment]
there was a separate issue around this when we introduced moving views because the title of the panel is cached and we cannot calculate the actual title until after the view container is opened. @sandy081 and I can take another look but due to when things load it wasn't a trivial bug to fix but more an accepted design issue.

**Comment by sbatten** [comment]
I will take a look after testing to see if there is something to be done about language changes specifically

**Comment by sandy081** [comment]
Good catch. @sbatten you are right and we shall consider language change while retrieving from cache.
